website/docs/r/recovery_services_vault.html.markdown

```diff
@@ -58,6 +58,8 @@ The following arguments are supported:
 
 * `soft_delete_enabled` - (Optional) Is soft delete enable for this Vault? Defaults to `true`.
```

Comment:
I may suggest implementing the `optional` to `required` change with `features.FivePointOh()` and put it to the `breaking-changes.md`